CollAction/Models/AdminViewModels/ManageProjectViewModel.cs

```diff
@@ -69,7 +69,7 @@ public class ManageProjectViewModel
 
         [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. 'tag1;tag2'")]
         [MaxLength(30)]
-        [RegularExpression(@"^#[a-zA-Z_]\w*(;#[a-zA-Z_]+\w*)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
+        [RegularExpression(@"^[a-zA-Z_]+(;[a-zA-Z_]+)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
```

Comment:
Ah, the intention was no longer need to enter "#" when entering tags.

Got very frustrated entering tags, suddenly noticed the need to enter #-es. Bit counter-intuitive. Gonna correct this by allow digits. Would want to allow things allow #savetheforest2015.

We'll need a tag entering UI anyway, this is just an in-between solution.